test/parallel/test-cluster-fork-env.js

```diff
@@ -56,10 +56,12 @@ if (cluster.isWorker) {
   });
 
   process.once('exit', function() {
-    assert.ok(checks.using, 'The worker did not receive the correct env.');
+    assert.ok(checks.using, 'The worker did not receive the correct env.' +
+    `${checks.using}`);
     assert.ok(
       checks.overwrite,
-      'The custom environment did not overwrite the existing environment.');
+      'The custom environment did not overwrite the existing environment.' +
+      `${checks.overwrite}`);
```

Comment:
It seems we are lacking spaces here and using unnecessary concatenation and redundant templates. What if we use something like this?
```js
    assert.ok(
      checks.using,
      `The worker did not receive the correct env. ${checks.using}`);
    assert.ok(
      checks.overwrite,
      `The custom environment did not overwrite the existing environment. ${
        checks.overwrite}`);
```